### PR TITLE
Review pages for in-context language issues

### LINKED ISSUE #211: Move symptom translations to front end
Right now, symptoms are translated in the back end (`statistics.ts:17`). I believe this should be moved to the front end.

```js
const symptomToLabelMap = {
  [Symptom.DRY_COUGH]: 'Tørrhoste',
  [Symptom.EXHAUSTION]: 'Slitenhet eller utmattelse',
  [Symptom.FEVER]: 'Feber',
  [Symptom.HEAVY_BREATHING]: 'Tung pust',
  [Symptom.MUSCLE_ACHING]: 'Muskelsmerter',
  [Symptom.DIARRHEA]: 'Diaré',
  [Symptom.HEADACHE]: 'Hodepine',
  [Symptom.SORE_THROAT]: 'Vondt i halsen',
  [Symptom.NO_TASTE]: 'Tap av smakssans',
  [Symptom.NO_SMELL]: 'Tap av luktesans',
  [Symptom.SLIME_COUGH]: 'Slimhoste',
  [Symptom.RUNNY_NOSE]: 'Tett eller rennende nese'
};
```

### LINKED ISSUE #240: Date time translation not correct
afbeelding

### LINKED ISSUE #246: Dutch site should mention post code should be without characters
Dutch postal codes are in the form 0000AA, but the AA is not needed.

Possibly for the dutch site only, is there a way to do this neatly? (we could do if statements in the template but that doesn't sound great. @fossecode ideas?)

### PR DESCRIPTION
**Pull request by Kilian**
Going through pages in context to see if translations make sense. Ready for review.

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -1,146 +1,147 @@
...
-  "API": "API",
-  "https://who.int": "https://who.int",
-  "To WHO.int": "To WHO.int",
-  "Are you from": "Are you from",
-  "Norway": "Norway",
-  "Netherlands": "The Netherlands",
-  "Slovakia": "Slovakia",
-  "United Kingdom": "The United Kingdom"
-}
+	"Postal code": "Postal code",
```
Comment (fossecode) [inline app/locales/en.json]: Tabs :scream:

Comment (Kilian) [reply app/locales/en.json]: normalised.

**Comment by Kilian** [comment]
Ready for review. Also fixes #211 and #240.

File: app/locales/nl.json [hunk 2]
```diff
@@ -1,146 +1,150 @@
...
-  "Symptom": "Symptoom",
-  "Added to form": "Toegevoegd aan formulier",
-  "https://who.int": "https://who.int",
-  "To WHO.int": "To WHO.int",
-  "Are you from": "Kom je uit",
-  "Norway": "Noorwegen",
-  "Netherlands": "Nederland",
-  "Slovakia": "Slowakije",
-  "United Kingdom": "Verenigd Koninkrijk"
+	"Postal code": "Postcode",
```
Comment (fossecode) [inline app/locales/nl.json]: Could we change this back to using spaces?

File: app/views/pages/statistics.ejs [hunk 3]
```diff
@@ -160,10 +160,11 @@
       /* Infected symptom count */
       var ctx = document.getElementById('infectedSymptomsChart').getContext('2d');
       var infectedSymptoms = <%- JSON.stringify(locals.infectedStats.symptomStats) %>;
+      var infectedSymptomslabels = <%- JSON.stringify(locals.infectedStats.symptomStats.labels.map(label => __(label))) %>;
```
Comment (fossecode) [inline app/views/pages/statistics.ejs:163]: Nice!

File: app/locales/nl.json (hunk 2 above)
Comment (Kilian) [reply app/locales/nl.json]: Yes, sorry.

**Review by fossecode — APPROVED** [review]

**Pull request merged**